[TensorPipe] Prepare for new Pipe API.

beauby opened the pull request:
Test Plan: CI

Reviewed By: lw

Differential Revision: D27648291

mrshenli reviewed (approved):
LGTM! This is mostly replacing TpMessage with TpDescriptor, where the latter is a alias of the former, right?

beauby commented:
Correct, this PR simply renames variables and makes them use the type aliases (that will become actual types in a subsequent PR).

Commit:
[TensorPipe] Prepare for new Pipe API. (#55820)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/55820

Test Plan: CI

Reviewed By: lw

Differential Revision: D27648291

fbshipit-source-id: 2fdb03f3954753048236ee50579ba1061cdcda2c
